### ISSUE TITLE
Typescript `import type` statements are not sorted the same way in v4.0.0 and v4.1.0

### ISSUE THREAD

**Issue by acelaya** [trivago/prettier-plugin-sort-imports#211]
**Your Environment**
 * **Prettier version**: 2.8.4
 * **node version** [18.4.2]:
 * **package manager**: [yarn@1]
 * **IDE**: [VScode, Webstorm, CLI]

**Describe the bug**

We have just updated to v4.1.0 of this library, and now it tries to sort all `import type` statements together at the top of the imports block.

Based on [the diff](https://github.com/trivago/prettier-plugin-sort-imports/compare/v4.0.0...v4.1.0#diff-69d46a8e0538df294b9b89ceec080451c586c065ff441c585f7d8fd57bbf2f3dR20) between v4.0 and v4.1, this seems like an intended behavior, and there's even mentions to new `<THIRD_PARTY_TS_TYPES>` and `<TS_TYPES>` tokens for `importOrder` option.

However, this is a breaking change, so I would expect it to be an opt-in behavior, or at least to provide an option to keep the previous behavior.

Would you consider providing such option to retain previous behavior?

**To Reproduce**

This script can be used to experiment the behavior:

```ts
import type { Component } from '@bar/bar';
import classnames from 'classnames';

import type { Foo } from '../aaa';
import Menu from './Menu';
import MenuItem from './MenuItem';
```

Running `prettier --write` on the file above using the configuration provided below, results in no changes when using v4.0.0

Using v4.1.0, it results in this:

```ts
import type { FilterOption } from '../aaa';
import type { Component } from '@bar/bar';
import classnames from 'classnames';

import Menu from './Menu';
import MenuItem from './MenuItem';
```

**Expected behavior**

The plugin keeps the same behavior, and it's safe to update from v4.0.0 to v4.1.0. Behavior changes and new features are observed only when opting-in to new features.

**Configuration File (cat .prettierrc, prettier.config.js, .prettier.js)**

Our prettier config:

```json
{
  "arrowParens": "avoid",
  "singleQuote": true,
  "importOrder": ["^[./]"],
  "importOrderSeparation": true
}
```

**Contribute to @trivago/prettier-plugin-sort-imports**

- [x] I'm willing to fix this bug 🥇

**Comment by ayusharma** [comment]
Hi @acelaya, we mistakenly release a new breaking change feature (#153) in v4.1.0. I will release a fix now.

**Comment by acelaya** [comment]
I see! Thanks for the quick response :)

**Comment by ayusharma** [comment]
Fixed released in v4.1., @acelaya Could you please verify the fix ?

**Comment by acelaya** [comment]
I was just testing it. Everything back to how it was before. Thanks for the super quick fix 🙂
